db/data/regions/_default/en.yml

```diff
@@ -146,75 +200,97 @@ en:
+            contains_url: City can't contain a URL
+            unknown_for_zip_instructional: Select a valid city for %{zip}
+            unknown_for_zip_informative: City doesn't match %{zip}
+            unknown_for_province_instructional: Enter a valid city for %{province}
+            unknown_for_province_informative: Province match %{province}
+            unknown_for_zip_and_province_instructional: Enter a valid city for %{zip}, %{province}
+            unknown_for_zip_and_province_informative: City may be incorrect
+            unknown_for_address: City may be incorrect
         province:
           label:
             default: Region
             optional: Region (optional)
           errors:
-            blank: Select a region
-            invalid: Enter a valid region
-            invalid_for_country: Enter a valid region for %{country}
-            unknown_for_city_and_zip: Enter a valid region for %{city}, %{zip}
-            unknown_for_city: Select a valid region for %{city}
-            unknown_for_zip: Select a valid region for %{zip}
-            unknown_for_address: Region may be incorrect.
+            blank_instructional: Select a region
+            blank_informative: Region is missing
+            invalid_instructional: Select a region
+            invalid_informative: Region is missing
+            invalid_for_country_instructional: Enter a valid region for %{country}
+            invalid_for_country_informative: Region doesn't match %{country}
+            unknown_for_city_and_zip_instructional: Enter a valid region for %{city}, %{zip}
+            unknown_for_city_and_zip_informative: Region doesn't match %{city}, %{zip}
+            unknown_for_city_informative: Select a valid region for %{city}
+            unknown_for_city_instructional: Region doesn't match %{city}
+            unknown_for_zip_instructional: Select a valid region for %{zip}
+            unknown_for_zip_informative: Region doesn't match %{zip}
+            unknown_for_address: Region may be incorrect
         country:
           label:
             default: Country/region
             optional: Country/region (optional)
           errors:
-            blank: Select a country/region
-            not_supported: We don't currently support this country/region. Enter a
-              new address and try again.
-            invalid: Enter a valid country/region
-            invalid_for_zip: Enter a valid country/region for this postal code
+            blank_instructional: Select a country/region
+            blank_informative: Country/region is missing
+            not_supported_instructional: Country/region not supported. Enter a new address and try again
+            not_supported_informative: Country/region not supported
+            invalid_instructional: Enter a valid country/region
+            invalid_informative: Country/region isn't valid
+            invalid_for_zip_instructional: Enter a valid %{country} for this country/region
```

Comment:
Seems like this should be `Enter a valid country/region for this %{zip}`, perhaps?